src/main/scala/nodes/images/FisherVector.scala

```diff
@@ -64,7 +64,7 @@ case class FisherVector(gmm: GaussianMixtureModel)
  */
 case class ScalaGMMFisherVectorEstimator(k: Int) extends Estimator[DenseMatrix[Float], DenseMatrix[Float]] {
   def fit(data: RDD[DenseMatrix[Float]]): FisherVector = {
-    val gmmTrainingData = data.flatMap(x => convert(MatrixUtils.matrixToColArray(x), Double))
```

Comment:
Looking back I'm surprised Breeze ever let convert(Array[DenseVector[...]]) work in the first place